.github/workflows/stale.yml

```
@@ -0,0 +1,29 @@
+name: Close stale issues and PRs
+
+on:
+  schedule:
+    # run weekly on sunday
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 14
```

```suggestion
          days-before-issue-stale: 30
```
2 weeks is pretty short. I'd say a month minimum, *maybe* more?